cinema/views.py

```diff
@@ -127,6 +123,28 @@ def upload_image(self, request, pk=None):
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "title",
+                type={"type": "list", "items": {"type": "number"}},
+                description="Filter by title (ex. ?title=Looper)"
+            ),
+            OpenApiParameter(
+                "genres",
+                type={"type": "list", "items": {"type": "number"}},
+                description="Filter by genres id (ex. ?genre=2,4)"
+            ),
+            OpenApiParameter(
+                "actors",
+                type={"type": "list", "items": {"type": "number"}},
+                description="Filter by actors id (ex. ?actor=1,3)"
+            ),
+        ]
+    )
+    def list(self, request, *args, **kwargs):
+        return super().list(self, request, *args, **kwargs)
+
 
 class MovieSessionViewSet(viewsets.ModelViewSet):
```

Comment:
what about adding filtering params to swagger here ?
![image](https://github.com/mate-academy/py-documentation-and-tests/assets/103201010/1b947061-8d59-45cb-8202-d2195d59a9b6)